src/main/java/org/sonarsource/scanner/maven/bootstrap/ScannerBootstrapper.java

```diff
@@ -38,12 +40,16 @@
 import org.sonarsource.scanner.api.EmbeddedScanner;
 import org.sonarsource.scanner.api.ScanProperties;
 
+import static org.sonarsource.scanner.maven.bootstrap.MavenProjectConverter.getPropertyByKey;
+
 /**
  * Configure properties and bootstrap using SonarQube scanner API
  */
 public class ScannerBootstrapper {
 
   static final String UNSUPPORTED_BELOW_SONARQUBE_56_MESSAGE = "With SonarQube server prior to 5.6, use sonar-maven-plugin <= 3.3";
+  private static final String SONAR_HOST_URL_KEY = "sonar.host.url";
```

Comment:
We can replace this one with `ScannerProperties.HOST_URL` that is already used elsewhere in the code